### PR TITLE
Added Documentation regarding Other related Repositories

### LINKED ISSUE #1547: No Documentation Regarding Other Related Repositories
No Documentation Regarding Other Related Repositories such as,

1. https://github.com/fossasia/open-event  
2. https://github.com/fossasia/open-event-orga-server 
3. https://github.com/fossasia/open-event-android

### PR DESCRIPTION
**Pull request by naveentherocker**
Fixes #1547 
Added Documentation with the links of,

1. https://github.com/fossasia/open-event
2. https://github.com/fossasia/open-event-orga-server
3. https://github.com/fossasia/open-event-android

### CODE REVIEW

**Comment by naveentherocker** [comment]
@fossasia/open-event-web-app-admin Please Review this

File: README.md [hunk 1]
```diff
@@ -100,6 +100,31 @@ The HTML pages of the generated website are created using Handelbars templates.
  - **[eventPage.js](/src/selenium/eventPage.js)** - *Contains methods for testing of the index page of the event*
  - **[trackPage.js](/src/selenium/trackPage.js)** - *Contains methods for testing of the tracks page of the event*
 
+## Support to Other Related Repositories
```
Comment (shubham-padia) [inline README.md]: Rename to `Other related repositories`

File: README.md [hunk 2]
```diff
@@ -100,6 +100,31 @@ The HTML pages of the generated website are created using Handelbars templates.
...
+
+### Open Event Format Definition
+
+> The **Open Event Project** enables the exchange of data between all components as well as with other services through a standardized Format. The Open Event repository provides a sample implementation of the format. It includes `JSON` files for all relevant event information and binary data for images and audio files.
+
+#### Repository: https://github.com/fossasia/open-event   
+
+#### Chat Channel: https://gitter.im/fossasia/open-event
+
+### Open Event Organizer Server
```
Comment (shubham-padia) [inline README.md]: Replace open event organizer server with `open event server`

**Comment by naveentherocker** [comment]
@shubham-padia Done

**Comment by naveentherocker** [comment]
Codacy was unable to analyze your pull request @isuruAb

**Comment by naveentherocker** [comment]
@isuruAb It is because of Failure in Travis Build.

**Comment by naveentherocker** [comment]
It's better if you could merge it @shubham-padia

**Comment by naveentherocker** [comment]
@geekyd @shubham-padia Please Merge it if you can

**Pull request merged**